frontend/app/routes/$lang+/_protected+/alerts+/index.tsx

```diff
@@ -17,15 +16,19 @@ export async function loader({ context: { session }, params, request }: LoaderFu
   await raoidcService.handleSessionValidation(request, session);
 
   const userInfoToken: UserinfoToken = session.get('userInfoToken');
+
   invariant(userInfoToken.sin, 'Expected userInfoToken.sin to be defined');
 
-  const alertSubscription = await getSubscriptionService().getSubscription(userInfoToken.sin);
+  //const alertSubscription = await getSubscriptionService().getSubscription(userInfoToken.sin);
```

Comment:
Should this be removed?

Reply:
We should still have the service call to get the subscriptions for the user here. The intent of this route is to redirect based on the status of the subscription.

Reply:
There is a TODO kind of indicating the above, so I will resolve.